src/packages/media/media/property-editors/image-crops-configuration/property-editor-ui-image-crops-configuration.element.ts

```diff
@@ -101,6 +107,24 @@ export class UmbPropertyEditorUIImageCropsConfigurationElement
 			: html`<uui-button look="secondary" type="submit" label="Add"></uui-button>`;
 	}
 
+	#onLabelInput() {
+		const value = this._labelInput.value ?? '';
+
+		const aliasValue = value.toLowerCase().replace(/[^a-z0-9]/g, '-');
```

Comment:
We have a small util function to generate an alias. Can we use the same here?

You can see how it is used in: https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/main/src/packages/core/components/input-with-alias/input-with-alias.element.ts#L6